src/components/AutocompleteLocation.vue

```diff
@@ -0,0 +1,36 @@
+<template>
+    <div class="columns">
+    <GmapPlaceInput
```

Comment:
it is but for better understanding what this mount do it has it's own name: AutocomplateLocation.vue